packages/api/src/routes/internal/medical/patient.ts

```diff
@@ -1156,4 +1161,61 @@ router.delete(
   })
 );
 
+/**
+ * POST /internal/patient/:id/recreate-consolidated
+ *
+ * Forcefully recreates the consolidated bundle for a patient.
+ *
+ * @param req.query.cxId The customer ID.
+ * @param req.params.id The patient ID.
+ */
+router.post(
+  "/:id/recreate-consolidated",
+  handleParams,
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const id = getFromParamsOrFail("id", req);
+
+    const patient = await getPatientOrFail({ id, cxId });
+
+    const requestId = uuidv7();
+    const startedAt = new Date();
+
+    const progress: ConsolidatedQuery = {
+      requestId,
+      status: "processing",
+      startedAt,
+    };
+
+    await storeQueryInit({
```

Comment:
Why are we updating the query status? To reset it in case it was borked, to coordinate sending WH, something else, all of the above?